app/config/services.yml

```diff
@@ -24,6 +24,11 @@ services:
             - { name: doctrine.event_listener, event: prePersist, method: prePersist }
             - { name: doctrine.event_listener, event: preRemove, method: preRemove }
 
+    fixDefaultSchemaListener:
+        class: AppBundle\EventListener\FixDefaultSchemaListener
+        tags:
+            - { name: doctrine.event_listener, event: postGenerateSchema, method: postGenerateSchema }
+
```

Comment:
I think it's good to avoid a giant `services.yml`. You'll see I've gradually created service files in the `app/services` directory, and import each one from `services.yml`. It might be a good idea to create an `event_listeners.yml`.

Also, in keeping with the naming convention of the services in those new files, we could name this service `AppBundle\EventListener\FixDefaultSchemaListener` which I think also helps with auto wiring (although not strictly relevant here).

Reply:
Good idea; I moved the other listeners in there too 👍